examples/swagger-to-ballerina/swagger_to_ballerina.yaml

```diff
@@ -5,7 +5,7 @@ info:
   license:
     name: MIT
 servers:
-  - url: http://petstore.swagger.io/v1
+        - url: http://localhost:8080/petstore/v1
```

Comment:
Shall we use 9090 here. We use port 9090 in ballerina examples as the default port.

Reply:
Done